apps/web/src/components/dashboard/energy-cost-card.tsx

```diff
@@ -43,60 +42,45 @@ export default async function EnergyCostCard({ startDate, endDate }: Props) {
     const userData = await getUserDataHistory(userId);
 
     const rawCosts = calculateCosts(userData, energyData);
-    const costString = rawCosts === 0 ? null : rawCosts.toFixed(2);
-    const cost = costString ? parseFloat(costString) : null;
+    const cost = rawCosts.toFixed(2);
 
     const calculatedPayment = getCalculatedPayment(userData, startDate, endDate);
-    const predictedCost = (cost ?? 0) + getPredictedCost(userData, energyData);
-
+    const predictedCost = getPredictedCost(userData, energyData);
+    const formattedCost = parseFloat(cost).toLocaleString('de-DE', { minimumFractionDigits: 2, maximumFractionDigits: 2 });
+    const formattedPredictedCost = predictedCost.toLocaleString('de-DE', { minimumFractionDigits: 2, maximumFractionDigits: 2 });
+    const forecastMonth = format(new Date(), "MMMM yyyy", {locale: de});
+    
     return (
         <Card className="w-full">
             <CardHeader>
                 <CardTitle>Energiekosten</CardTitle>
                 <CardDescription>
                     {startDate.toDateString() === endDate.toDateString() ? (
-                        <>
-                            {format(startDate, "PPP", {
-                                locale: de,
-                            })}
-                        </>
+                        <>{format(startDate, "PPP", {locale: de})}</>
                     ) : (
                         <>
-                            {format(startDate, "PPP", {
-                                locale: de,
-                            })}{" "}
-                            -{" "}
-                            {format(endDate, "PPP", {
-                                locale: de,
-                            })}
+                            {format(startDate, "PPP", {locale: de})} - {format(endDate, "PPP", {locale: de})}
                         </>
                     )}
                 </CardDescription>
             </CardHeader>
             <CardContent>
-                {cost !== null ? (
+                {parseFloat(cost) > 0 ? (
                     <>
-                        <h1 className="text-center text-2xl font-bold text-primary">{cost} €</h1>
-                        <p
-                            className={`text-center ${Number(cost) > Number(calculatedPayment) ? "text-red-500" : "white"}`}
-                        >
-                            Abschlag: {calculatedPayment} €
+                        <h1 className="text-center text-2xl font-bold text-primary">{formattedCost} €</h1>
+                        <p className={`text-center ${parseFloat(cost) > parseFloat(calculatedPayment || '0') ? "text-red-500" : "text-primary"}`}>
+                            Abschlag: {parseFloat(calculatedPayment || '0').toLocaleString('de-DE', { minimumFractionDigits: 2, maximumFractionDigits: 2 })} €
```

Comment:
`parseFloat(cost)` und `parseFloat(calculatedPayment || '0')` könntest du jeweils noch in Variablen auslagern, damit man das zum einen nicht mehrfach berechnen muss und zum anderen der Code lesbarer wird